.github/workflows/push-release.yml

```diff
@@ -0,0 +1,31 @@
+name: any-pr
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  push-master:
+    # Only run on any commit that features the release commit message syntax in the commit message
+    if: "contains(github.event.head_commit.message, 'release commit')"
```

Comment:
Note that when a "release commit" is pushed to master, both the unstable and the stable version will be published (unless "skip ci" is also present) Is that the desired behavior?

Reply:
Correct and see the build script for the commit message that is created when a release commit is made `[skip ci]` is included

Reply:
To be clear I would like to see us move to more standard commit message formats anyway e.g see #775 which would shift this convention to be more like `chore(release): publish [skip ci]`